### PR TITLE
was returning null values; tests break cause of no www

### PR DESCRIPTION
**Pull request by kevinchern**
- getResourceAsStream was returning null values, see https://stackoverflow.com/questions/16570523/getresourceasstream-returns-null
- tests started failing (I guess since last department website change?) because https://stat.ubc.ca/~bouchard/pub/ doesn't work, need https://www.stat.ubc.ca/~bouchard/pub/

### CODE REVIEW

**Comment by kevinchern** [comment]
Once this goes through, I think we need to update [blangDemos](https://github.com/UBC-Stat-ML/blangDemos) and [conifer](https://github.com/UBC-Stat-ML/conifer/) to use this version.

File: src/main/java/briefj/BriefIO.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ public ResourceCharSource(String url, Charset charset)
     @Override
     public Reader openStream() throws IOException
     {
-      InputStream resourceStream = new Object().getClass().getResourceAsStream(url);
+      InputStream resourceStream = this.getClass().getResourceAsStream(url);
```
Comment (alexandrebouchard) [inline src/main/java/briefj/BriefIO.java:168]: I don't understand this change. Why is this necessary?

**Review by alexandrebouchard — COMMENTED** [review]
I suggest changing these tests as they are fragile as follows:
- let us move the files to the github repo
- let us use the github URL instead

As a second thing, it would be good to make sure the methods in question are well behaved when the URL is not working - e.g. an exception documented in the method.
